recipes/batchcrop/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "batchcrop" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "42b7476326ac0ef0ac7910b82a70ac1c073035237813790d6e29625a003421db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/batchcrop/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py<33]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - batchcrop = batchcrop.cli.batchcrop:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - docopt
+  run:
+    - python
+    - docopt
+    - imagemagick
+
+test:
+  commands:
```

Comment:
Do you also want to test for "import batchcrop" ?

Reply:
Thanks! Done!